interpreters/wamr/Kconfig

```diff
@@ -112,9 +112,17 @@ config INTERPRETERS_WAMR_THREAD_MGR
 	bool "Enable thread manager"
 	default n
 
+config INTERPRETERS_WAMR_LIB_WASI_THREADS
```

Comment:
should we use choice here?

Reply:
no.
you can enable both of them if you want.